scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+                            record_count += 1
+                            
+                            if record_count % 10000 == 0:
+                                temp_file.flush()
+                        except Exception as e:
+                            continue
+        
+        temp_file.close()
+        gc.collect()
+        return prefecture, temp_path, record_count
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)